ops/testing.py

```diff
@@ -446,16 +444,19 @@ def _next_relation_id(self):
         self._relation_id_counter += 1
         return rel_id
 
-    def add_storage(self, storage_name: str, count: int = 1) -> typing.List[str]:
-        """Declare a new storage device attached to this unit.
+    def add_storage(self, storage_name: str, count: int = 1,
+                    attach: bool = False) -> typing.List[str]:
```

Comment:
Should this be restricted to kw only?

```suggestion
                    *, attach: bool = False) -> typing.List[str]:
```